src/shared/firewall-util-nft.c

```diff
@@ -937,6 +940,49 @@ static int nft_set_element_op_in_addr(
         return sd_nfnl_call_batch(nfnl, &m, 1, NFNL_DEFAULT_TIMEOUT_USECS, NULL);
 }
 
+int nft_set_element_op_in_addr_open(
+                FirewallContext **ctx,
+                bool add,
+                const NFTSetContext *nft_set_context,
+                int af,
+                const union in_addr_union *address,
+                unsigned int prefixlen) {
+        const char *table, *set;
+        int r, nfproto;
+
+        assert(ctx);
+
+        if (!*ctx) {
+                r = fw_ctx_new(ctx, /* init_tables = */ false);
+                if (r < 0)
+                        return r;
+        }
+
+        assert((*ctx)->nfnl);
+        assert(nft_set_context);
+        assert(address);
+        nfproto = nft_set_context->nfproto;
+        table = nft_set_context->table;
+        assert(table);
+        set = nft_set_context->set;
+        assert(set);
+
+        r = nft_set_element_op_in_addr((*ctx)->nfnl, add, nfproto, af, table, set, address, prefixlen);
+
```

Comment:
please remove this empty line (we try to keep function calls and their immediate error handling together)

Reply:
ok

Reply:
argh, forgot this

Reply:
done

Reply:
done

Reply:
done